rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -940,6 +941,31 @@ void saveNextFederationCreationBlockHeight_postIrisAndHeightIsNotNull_shouldSave
 
     }
 
+    @Test
+    void saveLastRetiredFederationP2SHScript_postIris_getsValueFromStorage() {
+
+        // Arrange
+
+        ActivationConfig.ForBlock irisActivations = ActivationConfigsForTest.iris300().forBlock(0L);
+
+        Script expectedScript = new P2shErpFederationBuilder().build().getP2SHScript();
```

Comment:
```suggestion
        Script expectedScript = new P2shErpFederationBuilder().build().getDefaultP2SHScript();
```
Let's use that one that is the one actually stored

Reply:
Updated.